xdsl/interpreter.py

```diff
@@ -362,25 +362,6 @@ def run_block(self, block: Block, args: PythonValues) -> PythonValues | None:
         The terminator of this block is expected either to call its successor or return
         the results for the region directly.
         """
```

Comment:
Is this git diffs messing with me or is this function now just empty? 🤔

Reply:
Hmm I'll check the whole function should be deleted